packages/core/src/app/Renderer.ts

```diff
@@ -298,12 +304,14 @@ export class Renderer {
     if (
       result === RendererResult.Success &&
       this.exporter &&
-      this.exporter.generateAudio
+      this.exporter.mergeMedia
     ) {
-      //only generate audio when rendering images was actually successful
-      const endFrame = Math.min(this.playback.duration, this.playback.frame);
+      //only merge media when rendering images was actually successful
```

Comment:
Minor nitpick haha: comments describing a conditional usually go before the if statement and not inside. Also I think it'd be cool if we honored the commenting style of of the rest of the codebase, i.e.  capitalized, with a space between the slashes and the first word and with a period in the end. 
`// Only merge ... successful.`